src/internal/easee/thing.go

```diff
@@ -111,17 +116,25 @@ func (t *thingFactory) inclusionReport(info *Info, thingState adapter.ThingState
 	}
 }
 
-func (t *thingFactory) chargepointSpecification(adapter adapter.Adapter, thingState adapter.ThingState, groups []string, state *State) *fimptype.Service {
-	return chargepoint.Specification(
-		adapter.Name(),
-		adapter.Address(),
-		thingState.Address(),
-		groups,
-		t.supportedStates(),
+func (t *thingFactory) chargepointSpecification(ad adapter.Adapter, thingState adapter.ThingState, groups []string, state *State) *fimptype.Service {
+	options := []adapter.SpecificationOption{
 		chargepoint.WithChargingModes(SupportedChargingModes()...),
 		chargepoint.WithPhases(state.Phases),
```

Comment:
Phases, grid type and supported max current should also be added conditionally (phases > 0, max current > 0, grid type defined).